nixos/modules/services/networking/technitium-dns-server.nix

```diff
@@ -0,0 +1,130 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
Don't use top-level `with lib`, use `inherit` instead.

Reply:
Thank you for your review. I hadn't seen that before. Fixed it.